java/gazelle/private/servermanager/insource-server.go

```diff
@@ -0,0 +1,37 @@
+package servermanager
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/bazelbuild/rules_go/go/runfiles"
+)
+
+func javaparserLocation() string {
+	return "contrib_rules_jvm/java/src/com/github/bazel_contrib/contrib_rules_jvm/javaparser/generators/Main"
+}
+
+func (m *ServerManager) locateJavaparser(jvmFlags []string) (string, error) {
+	rf, err := runfiles.New()
+	if err != nil {
+		return "", fmt.Errorf("failed to init new style runfiles: %w", err)
+	}
+
+	javaparserPath := javaparserLocation()
+	if runtime.GOOS == "windows" {
+		javaparserPath += ".exe"
+	}
```

Comment:
This exists twice, could we reasonably DRY it?

Reply:
I'm not sure, because while the condition is the same as `embedded.go`, the effect is different. In embedded.go, we definitely want a `.bat` file, because it's a script and not an executable. Happy to store a boolean in `ServerManager`, but it doesn't feel like something the servermanager should care about.